orangecontrib/spectroscopy/tests/example_connect_settings.py

```diff
@@ -8,13 +8,13 @@
 import sys
 from decimal import Decimal
 
-import Orange
-from Orange.widgets.widget import OWWidget
-
 from orangecontrib.spectroscopy.widgets.gui import connect_settings, lineEditFloatRange, \
     ValueTransform, connect_line, MovableVline
 from orangecontrib.spectroscopy.widgets.owspectra import CurvePlot
 
+import Orange
+from Orange.widgets.widget import OWWidget
+
```

Comment:
Why did you move imports from Orange? Did the linter complain?

From PEP-0008:

```
Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.
```

Reply:
Yeah, I get lots of these: 

> C0411: third party import "from orangecontrib.spectroscopy.preprocess import Interpolate, interp1d_with_unknowns_numpy, interp1d_with_unknowns_scipy, interp1d_wo_unknowns_scipy, InterpolateToDomain, NotAllContinuousException" should be placed before "import Orange" (wrong-import-order)

Reply:
Ah, so in this case, pylint does not know that Orange is a third party import. Maybe we would need to fix lint config (probably it reads something from setup.py).